### ISSUE TITLE
Are the reported results on the validation set or the training set?

### ISSUE THREAD

**Issue by abduallahmohamed** [agrimgupta92/sgan#62]
Hi,

Are the reported results on the validation set or the training set?

Thanks
